railties/Rakefile

```diff
@@ -32,6 +32,7 @@ namespace :test do
 
     dirs = (ENV["TEST_DIR"] || ENV["TEST_DIRS"] || "**").split(",")
     test_files = dirs.map { |dir| "test/#{dir}/*_test.rb" }
+    testopts = ENV["TESTOPTS"].split(/[\s]+/)
```

Comment:
NoMethodError: undefined method `split' for nil:NilClass
https://travis-ci.org/rails/rails/jobs/398750204#L1166

Reply:
Thank you. Fixed, but perhaps you think it's appropriate to be more verbose than this ternary?